opm/output/eclipse/EclipseWriter.cpp

```diff
@@ -646,12 +625,7 @@ void EclipseWriter::writeTimeStep(int report_step,
        the conversion is done once here - and not closer to the actual
        use-site.
     */
-    convertFromSiTo( cells[ dc::PRESSURE ], units, UnitSystem::measure::pressure );
-    if ( cells.has( dc::TEMP ))
-        convertFromSiTo( cells[ dc::TEMP ],
-                         units,
-                         UnitSystem::measure::temperature );
-
+    cells.convertFromSI( units );
```

Comment:
Personally I don't like this kind of API because it's confusing and it's somewhat flaky (converting from SI twice is possible unless you keep internal state to make this method a no-op on successive calls). Can we lift the method and keep `cells` more oblivious to units?